modules/fedimint-mint-client/src/lib.rs

```diff
@@ -1274,49 +1307,22 @@ pub struct SpendableNote {
 }
 
 /// Base64 encode a set of e-cash notes. See also [`parse_ecash`].
-pub fn serialize_ecash(ecash: &TieredMulti<SpendableNote>) -> String {
+pub fn serialize_ecash(ecash: &OOBNotes) -> String {
     let mut bytes = Vec::new();
     Encodable::consensus_encode(ecash, &mut bytes).expect("encodes correctly");
     base64::encode(&bytes)
 }
 
 /// Decode a set of e-cash notes from a base64 string. See also
 /// [`serialize_ecash`].
-pub fn parse_ecash(s: &str) -> anyhow::Result<TieredMulti<SpendableNote>> {
+pub fn parse_ecash(s: &str) -> anyhow::Result<OOBNotes> {
```

Comment:
These functions should be transformed into `FromStr` and `Display` implementations now that there is a local type for serializable e-cash.